apps/wallet/src/ui/app/components/overlay/Overlay.module.scss

```diff
@@ -14,7 +14,8 @@
     position: fixed;
     z-index: 100;
     top: 0;
-    margin-left: -20px;
+    left: 0;
```

Comment:
I verified that the overlay sits the same on other screens, this should stretch the overlay more properly (unless I'm missing something with why the negative margin was added in the first place)